### ISSUE TITLE
Assign different gradient weights

### ISSUE THREAD

**Issue by Cris-chen** [calmisential/Basic_CNNs_TensorFlow2.0#24]
How to write the code to assign different gradient weights to different classes of SparseCategoricalCrossentropy

**Comment by calmiLovesAI** [comment]
The sample_weight param may work.  https://tensorflow.google.cn/api_docs/python/tf/keras/losses/SparseCategoricalCrossentropy#__call__
